recipes/sdsl-lite/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  cci.20210329:
+    url: https://github.com/xxsds/sdsl-lite/archive/163d59bb1f3b1b21b028099547f556953d1c626f.zip
+    sha256: 3ec5e85ee6caf5afe73c4077992aaaaffd5dfed711033e944937c6245adbfeca
+patches:
+  cci.20210329:
+    - patch_file: patches/cci.20210329/include/sdsl/bits.hpp.patch
```

Comment:
It's common to store this patch directly in the patches folder or in a `cci.20210329` subfolder if you expect each release to require a lot of patches.
When putting the directly in the `patches` folder, you can namespace the patches with a prefix.
e.g. `patches/cci.20210329-include-sdl-bits-hpp.patch`.

Reply:
if you prefer this, i can do, ofc, but i think this one represents way more the structure of patching